images/aquarium/config.xml

```diff
@@ -43,7 +43,7 @@
         </profile>
     </profiles>
     <preferences>
-        <version>16.0.0</version>
+        <version>0.1.0</version>
```

Comment:
This should ideally come from the latest version git tag, probably by renaming this file to images/aquarium/config.xml.in, changing this line to \<version\>VERSION\</version\> (or whatever), then adding an extra bit of script somewhere to s/VERSION/$(git describe [...])/, but I'm fine if this is in a separate PR.